boards/common/actinius/actinius_board_common.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2021-2022 Actinius
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(actinius_board_control,
+					CONFIG_ACTINIUS_BOARD_CONTROL_LOG_LEVEL);
+
+#define SIM_SELECT_NODE DT_NODELABEL(sim_select)
+#define CHARGER_ENABLE_NODE DT_NODELABEL(charger_enable)
+
+#if IS_ENABLED(CONFIG_ACTINIUS_BOARD_HAS_SIM_SELECT)
```

Comment:
just use `#ifdef CONFIG_...`, here and elsewhere. You only need IS_ENABLED when you need either 1 or 0 outside of an ifdef

Reply:
fixed, thanks!